### DISCUSSION TITLE
is it possible to draw multiple links in stencil paper

### DISCUSSION THREAD

**Discussion by vyshak-pr** [clientIO/joint#1969] [Q&A]
![wq](https://user-images.githubusercontent.com/120721238/210251466-9527b0ad-2193-447f-9d05-44570e4d9cf0.jpg)
is it possible to draw links like the above picture?
(in stencil paper I want different columns count)

**Comment by kumilingus** [comment]
Yes.
1. How to use links in stencil please see [this](https://github.com/clientIO/joint/discussions/1659) first.
2. You can disable the default stencil layout and position the elements in the stencil as you desire.
```js
const stencil = new ui.Stencil({
  /* ... */
  layout: false
});
stencil.load([
  { type: 'standard.Rectangle', position: { x: 10, y: 10 }, /* ... */ },
  { type: 'standard.Ellipse', position: { x: 10, y: 110 }, /* ... */ },
]);
```

**Reply by vyshak-pr** [reply]
@kumilingus Thanks
